### PR TITLE
[Doc Improvement][Add a link in see also of publish app overview][2809656]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3deae0e4369c383ab7f7fe27d8d6946b1c696b37/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-publish-overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8d232ee9-8896-4f91-85d7-abb4f82bf989) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md [hunk 1]
```diff
@@ -57,3 +57,4 @@ For more information, see [publish to the Teams store](~/concepts/deploy-and-pub
 * [Microsoft 365 App Compliance Program](/microsoft-365-app-certification/overview)
 * [Create Teams app package](../build-and-test/apps-package.md)
 * [Publish to org](/microsoftteams/upload-custom-apps)
+* [Modify the accounts supported by an application](/azure/active-directory/develop/howto-modify-supported-accounts)
```
Comment (ashishguptaiitb) [inline msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md:60]: Change the hyperlink text based on "_how to modify your app's registration in the Azure portal to change who, or what types of accounts, can access the application_" from the linked article.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/37d2bca3e7293b4d1f54a9bb2b11bc176c72b62c/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-publish-overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/77c645ae-603e-4a93-aa59-8c5cadfdd7fd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4363e8762d653bc31b5df8be9b3be74e8d40feba/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-publish-overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/def49363-7c53-4d20-b27d-80c9047fba51) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/459a1f3f0ad623def40587a0b7fda00292d79854/msteams-platform%2Fconcepts%2Fdeploy-and-publish%2Fapps-publish-overview.md) | 96 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ab577f8a-c7ef-4aff-a863-1e2dc5b2dae5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ashishguptaiitb — APPROVED** [review]

**Pull request merged**